library-best-practices/JLBP-17.md

```diff
@@ -0,0 +1,38 @@
+[JLBP-17] Coordinate Major Version Adoption
+-------------------------------------------
+
+It is clearly undesirable for a dependency tree to contain conflicts.
+Unfortunately, as soon as any library in a dependency tree introduces a breaking
```

Comment:
Suggested rewording:

When a library in a dependency tree introduces a breaking
change, consumers can't update to that version until all their other dependencies which use the library also update to that version

Is that a fair restatement?